src/Traits/ComparisonTrait.php

```diff
@@ -240,6 +240,66 @@ public function isTomorrow()
     }
 
     /**
+     * Determines if the instance is within the next week
+     *
+     * @return bool
+     */
+    public function isNextWeek()
+    {
+        return $this->toDateString() === static::now($this->tz)->addWeek()->toDateString();
```

Comment:
Would you mind changing this to avoid possible error in the future with timezones?

```php
static::now($this->tz) // Change to
$this->copy();
```